config/jobs/kubernetes-sigs/sig-windows/sig-windows-sac.yaml

```diff
@@ -1,6 +1,6 @@
 periodics:
 - interval: 72h
```

Comment:
Can we keep the SAC tests on master?

Reply:
1903 SAC is only valid for 1.18: https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#windows-os-version-support

Reply:
right, nevermind my comment then :)